support-frontend/assets/pages/paper-subscription-landing/components/content/deliveryTab.jsx

```diff
@@ -16,21 +16,8 @@ import {
 import { Collection } from 'helpers/productPrice/fulfilmentOptions';
 import { paperHasDeliveryEnabled } from 'helpers/subscriptions';
 import { Accordion, AccordionRow } from '@guardian/src-accordion';
-import { css } from '@emotion/core';
-import { neutral } from '@guardian/src-foundations/palette';
-import { textSans } from '@guardian/src-foundations/typography';
-import { space } from '@guardian/src-foundations';
 import { type Option } from 'helpers/types/option';
-
-
-const accordionContainer = css`
-  background-color: ${neutral['97']};
-
-  p {
-    ${textSans.small()};
-    margin-bottom: ${space[4]}px;
-  }
-`;
+import { accordionContainer } from './tabsStyles';
```

Comment:
My understanding is that inline css should never be shared, which I believe is pretty much a golden rule. I think this partly why they are usually kept in the same file as the component itself, so the temptation to share them is lessened! Given this, I think it would be best to keep them with their respective components.

Reply:
Makes sense!